snapcraft/internal/parts.py

```diff
@@ -273,7 +273,13 @@ def load_plugin(self, part_name, plugin_name, part_properties):
             part_schema=self._validator.part_schema,
             definitions_schema=self._validator.definitions_schema)
 
-        self.build_tools += part.code.build_packages
+        # Add multi-arch suffix for cross-compilation to dev packages
+        for package in part.code.build_packages:
+            if self._project_options.is_cross_compiling:
+                if package.endswith('-dev'):
```

Comment:
to be future proof and this will happen, we need to check for `:` to detect a hardcoding of an arch, an `all` entry or an `any`; in the case of no `:` or `any` I think is when we can append.